tasks/enable.yml

```
@@ -0,0 +1,100 @@
+---
+- name: Check trust compatibility
+  fail:
+    msg: Fapolicyd does not support trust setting fapolicyd_setup_trust
+  ignore_errors: true
+  when:
+    - fapolicyd_setup_trust | length > 0
+    - ansible_facts.distribution_version is version("8.2", "<=")
+  register: __failed_check_trust
+
+- name: Check integrity compatibility
+  fail:
+    msg: Fapolicyd does not support integrity setting fapolicyd_setup_integrity
+  ignore_errors: true
+  when:
+    - fapolicyd_setup_integrity | length > 0
+    - ansible_facts.distribution_version is version("8.3", "<=")
+  register: __failed_check_integrity
+
+- name: Check trust files compatibility
+  fail:
+    msg: Fapolicyd does not support trust files setting fapolicyd_add_trusted_file
+  ignore_errors: true
+  when:
+    - fapolicyd_add_trusted_file | length > 0
+    - ansible_facts.distribution_version is version("8.3", "<=")
+  register: __failed_check_trusted_file
+
+- name: Check failed conditions
+  fail:
+    msg: Multiple failed conditions
+  #failed_when: true
```

```suggestion
  # failed_when: true
```
yaml lint - requires space after `#` comment